File: Java/백준/Bronze/23348. 스트릿 코딩 파이터/스트릿 코딩 파이터.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        int b = in.nextInt();
        int c = in.nextInt();
        int t = in.nextInt();
        int max = -1;
        for (int i=0;i<t;i++){
            int total=0;
            for (int j=0;j<3;j++){
                int d = in.nextInt();
                int e = in.nextInt();
                int f = in.nextInt();
                total += a*d+b*e+c*f;
            }
            max = max>total?max:total;
        }
        System.out.println(max);
    }
}

File: C++11/백준/Bronze/23348. 스트릿 코딩 파이터/스트릿 코딩 파이터.cc
#include <iostream>
using namespace std;

int main(void){
    int a,b,c,d,e,f,max=-1,t;
    cin >> a >> b >> c >> t;
    for (int i=0;i<t;i++){
        int total = 0;
        for (int j=0;j<3;j++){
            cin >> d >> e >> f;
            total+=a*d+b*e+c*f;
        }
        max = max>total?max:total;
    }
    cout << max;
    return 0;
}